starter-code/bin/seeds-movies.js

```diff
@@ -0,0 +1,29 @@
+const movieModel = require("./../models/movie")
+const mongoose = require("mongoose")
+
+const moviesList = [{
+    title: 'Casino',
+    genre: 'Drama',
+    plot: 'some mafia stuff'
+}, {
+    title: 'Once Upon a Time in Hollywood',
+    genre: 'Drama, Comedy',
+    plot: 'some tarantino movie'
+}, {
+    title: 'Joker',
+    genre: 'Drama',
+    plot: 'DC Joker\'s story'
+}]
+
+mongoose
+    .connect('mongodb://localhost/starter-code', {
+        useNewUrlParser: true
+    })
+    .then(x => {
+        console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+    })
+    .catch(err => {
+        console.error('Error connecting to mongo', err)
+    });
+
+movieModel.create(moviesList).then(dbRes => console.log(dbRes)).catch(dbErr => console.log(dbErr))
```

Comment:
Better to have your seed this way, the connection to the DB is really fast so you didn't get any errors, but in a case the DB connection is slow, your create function might be quicker, hence leading to an error since the connection hasn't been established yet.
Same goes in the `seeds.js` file

```
mongoose
  .connect("mongodb://localhost/starter-code", {
    useNewUrlParser: true
  })
  .then(x => {
    console.log(
      `Connected to Mongo! Database name: "${x.connections[0].name}"`
    );

    movieModel
      .create(moviesList)
      .then(dbRes => console.log(dbRes))
      .catch(dbErr => console.log(dbErr));
  })
  .catch(err => {
    console.error("Error connecting to mongo", err);
  });

```